dataland-frontend/src/components/forms/parts/elements/basic/CheckboxesListFormElement.vue

```diff
@@ -73,7 +74,7 @@ export default defineComponent({
         this.shouldBeIgnored = false;
         this.checkboxValue = [currentValue];
```

Comment:
I removed it and it still seems to work.
Pls review this change on Monday if possible.

Reply:
This is to reflect currentValue in checkbox in edit mode to make it visible what the value is after loading the data
![image](https://github.com/d-fine/Dataland/assets/96523011/6bb3cb4f-c7ea-41e3-a3bd-14762ed650cc)

Reply:
I allowed myself to restoring this

Reply:
Ah ok I understand.  I made the mistake of thinking that `checkboxValue` in `updateCurrentValue()` and `this.checkboxValue` are the same variable, when they aren't.
The first one is the new value from the "input"-event, and the second one is a vue data variable that we created.

I renamed `checkboxValue` to `newCheckboxValue` in `updateCurrentValue()` to avoid that other devs do the same mistake as I did when reading the code.

Furthermore, I think I found a way to write the whole thing in less code while making it still work.

Can you check my last commit pls and tell me what you think?

Reply:
It's ok